src/pretix/presale/templates/pretixpresale/event/checkout_customer.html

```diff
@@ -8,134 +8,118 @@
     <form method="post">
         {% csrf_token %}
         <div class="panel-group" id="customer">
-            <div class="panel panel-default">
-                <div class="accordion-radio">
-                    <div class="panel-heading">
-                        <p class="panel-title">
+            <fieldset class="panel panel-default accordion-panel">
+                <legend class="accordion-radio">
+                    <label class="panel-heading">
+                        <span class="panel-title">
                             <input type="radio" name="customer_mode" value="login"
-                                    data-parent="#customer"
                                     {% if selected == "login" or not signup_allowed %}checked="checked"{% endif %}
-                                    id="input_customer_login"
-                                    data-toggle="radiocollapse" data-target="#customer_login"/>
+                                    id="input_customer_login" aria-controls="customer_login" />
                             <label for="input_customer_login"><strong>{% trans "Log in with a customer account" %}</strong></label>
```

Comment:
We already are inside a label :)
```suggestion
                            {% trans "Log in with a customer account" %}
```